test/examples/nix/sources.nix

```diff
@@ -3,13 +3,13 @@ let
   #
   # The fetchers. fetch_<type> fetches specs of type <type>.
   #
-  fetch_file = pkgs: name: spec: let
-    name' = sanitizeName name + "-src";
-  in
-    if spec.builtin or true
-    then
-      builtins_fetchurl
-      {
```

Comment:
Is it normal for this file to get reformatted?

Reply:
I don't know how that got changed. I reverted it back, and ran `nix fmt` to be sure there wasn't an issue with the config. It didn't get formatted again. Let's keep an eye in case something is off going forward. Good catch.